src/Category.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var Z = require('sanctuary-type-classes');
+
+var assert = require('./internal/assert');
+var compose = require('./internal/compose_');
+var id = require('./internal/id');
+
+
+module.exports = function(equals) {
+  return {
+
+    //  a `compose` id C = a
+    leftIdentity: assert.forall2(function(C, a) {
+      return Z.Category.test(a) &&
+             equals(compose(a)(id(C)),
+                    a);
+    }),
+
+    //  id C `compose` a = a
+    rightIdentity: assert.forall2(function(C, a) {
+      return Z.Category.test(a) &&
+             equals(compose(id(C))(a),
+                    a);
+    })
+
+  };
+};
```

Comment:
Thanks for checking, Gabe. It was this that looked wrong to me:

```console
$ ack --before-context 1 --no-filename -- leftIdentity src | awk 'NR % 3 == 1'
    //  a `compose` id C = a
    //  extend extract w = w
    //  pure x >>= f = f x
    //  empty <> m = m
    //  zero <|> x = x
```

In the cases of `<>` and `<|>` at least, the identity element is the left operand.